ci/test.sh

```diff
@@ -2,23 +2,21 @@
 python -m pip install --upgrade pip
 if [ -f tests/requirements.txt ]; then pip install -r tests/requirements.txt; fi
 
-# download <swagger.yaml> file
 cd /home/runner/work/lightning_py/lightning_py
-curl -O https://raw.githubusercontent.com/aerokube/selenium-openapi/master/selenium.yaml
 
 # install swagger codegen
 wget https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.34/swagger-codegen-cli-3.0.34.jar -O swagger-codegen-cli.jar
 
 java -jar swagger-codegen-cli.jar generate \
-   -i /home/runner/work/lightning_py/lightning_py/selenium.yaml \
+   -i https://raw.githubusercontent.com/aerokube/selenium-openapi/master/selenium.yaml \
    -l python \
-   -o /home/runner/work/lightning_py/lightning_py/python
-
+   -o /home/runner/work/lightning_py/lightning_py/generated \
+   -D packageName=lightning_adapter
 # setup swagger API
-pushd python/ && python setup.py install && cd ..
+pushd generated/ && python setup.py install && cd ..
```

Comment:
Need to do `popd` instead of `cd ..`

Reply:
fixed in 09b5f9cd6b08f41f9226ab61decbf54bb237b99b